configuration/raspi-config.md

```diff
@@ -135,51 +160,59 @@ Enable/disable shell and kernel messages on the serial connection.
 
 Enable/disable the Dallas 1-wire interface. This is usually used for DS18B20 temperature sensors.
 
+#### Remote GPIO
+
+Enable or disable remote access to the GPIO pins.
+
+### Performance Options
+
 <a name="overclock"></a> 
 ### Overclock
 
 It is possible to overclock your Raspberry Pi's CPU. The default is 700MHz but it can be set up to 1000MHz. The overclocking you can achieve will vary; overclocking too high may result in instability. Selecting this option shows the following warning:
```

Comment:
"The default is 700MHz" - nope, the default varies depending on the model of Pi you're using :wink:

Reply:
Ah, missed that!